src/frontend/Komponenter/Behandling/Sanksjon/Sanksjonsfastsettelse.tsx

```diff
@@ -131,26 +131,25 @@ const SanksjonsvedtakVisning: FC<{
     const internBegrunnelse = formState.getProps('internBegrunnelse') as FieldState;
 
     const håndterVedtaksresultat =
-        (nesteUrl: string): ((res: RessursSuksess<string> | RessursFeilet) => void) =>
+        (): ((res: RessursSuksess<string> | RessursFeilet) => void) =>
         (res: RessursSuksess<string> | RessursFeilet) => {
             if (res.status === RessursStatus.SUKSESS) {
-                navigate(nesteUrl);
+                utførRedirect();
                 hentBehandling.rerun();
-                nullstillIkkePersisterteKomponenter();
             } else {
                 settFeilmelding(res.frontendFeilmelding);
             }
         };
 
     const lagreVedtak = (vedtaksRequest: ISanksjonereVedtakForOvergangsstønad) => {
         settLaster(true);
-
+        nullstillIkkePersisterteKomponenter();
```

Comment:
Hvis kallet for å lagre feiler, så er komponenten nå så kan man gå videre uten å få opp varsel